### PR TITLE
workflow_lint: ensure that tool shed tool ids are valid

### PR DESCRIPTION
**Pull request by lldelisle**
This allows to check that if a workflows uses a tool in the toolshed it actually exists in the toolshed.
This is a follow up of #1305

### CODE REVIEW

File: planemo/workflow_lint.py [hunk 1]
```diff
@@ -46,6 +49,8 @@
 POTENTIAL_WORKFLOW_FILES = re.compile(r"^.*(\.yml|\.yaml|\.ga)$")
 DOCKSTORE_REGISTRY_CONF_VERSION = "1.2"
 
+AUTOUPDATE_TOOLSHED_URL = "https://toolshed.g2.bx.psu.edu"
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply planemo/workflow_lint.py:52]: maybe `MAIN_TOOSHED_URL` ?

[the remark this answers is not in the document]
Comment (lldelisle) [reply planemo/workflow_lint.py:52]: Yes good idea. Is it a good thing to define it only once and import it in autoupdate?

[the remark this answers is not in the document]
Comment (mvdbeek) [reply planemo/workflow_lint.py:52]: I think that's fine, yes. A future update could allow updating from any tool shed IMO, but this will do for now.

**Comment by lldelisle** [comment]
:smiley:

**Comment by mvdbeek** [comment]
Thanks so much, this is awesome!

**Comment by lldelisle** [comment]
I could not have done it by myself, thanks.

**Pull request merged**